src/easypost.js

```diff
@@ -12,6 +12,7 @@ import CarrierType, { propTypes as carrierTypePropTypes } from './resources/carr
 import Billing, { propTypes as BillingPropTypes } from './resources/billing';
 import CustomsInfo, { propTypes as customsInfoPropTypes } from './resources/customs_info';
 import CustomsItem, { propTypes as customsItemPropTypes } from './resources/customs_item';
+import EndShipper from './resources/end_shipper';
```

Comment:
Still need those prop types imported here and then used in the `PROP_TYPES` constant below.